specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-09-02-preview/managedClusters.json

```diff
@@ -7656,6 +7656,28 @@
           "type": "integer",
           "format": "int32",
           "description": "The maximum number of processes per pod."
+        },
+        "seccompDefault": {
+          "type": "string",
+          "description": "Specifies the default seccomp profile applied to all workloads. If not specified, 'Unconfined' will be used by default.",
+          "enum": [
+            "Unconfined",
+            "RuntimeDefault"
+          ],
+          "x-ms-enum": {
+            "name": "SeccompDefault",
+            "modelAsString": true,
+            "values": [
+              {
+                "description": "No seccomp profile is applied, allowing all system calls.",
+                "value": "Unconfined"
+              },
+              {
+                "description": "Container runtime's default seccomp profile is applied, which restricts certain system calls for enhanced security.",
```

Comment:
Please fix the failed [spell check](https://github.com/Azure/azure-rest-api-specs/pull/30974/checks?check_run_id=31387421298) by adding `runtime's` to words under category `"filename": "**/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/**/*json"` in file `cSpell.json`. Or change the description to something like `The default seccomp profile for container runtime is applied, ...`
![image](https://github.com/user-attachments/assets/623168e6-3617-42bf-a0a5-1de33c09c74d)